_drafts/2017-10-13-rprofile-david-smith.md

```diff
@@ -0,0 +1,119 @@
+---
+name: rprofile-david-smith
+layout: post_discourse
+title: ".rprofile: David Smith"
```

Comment:
@sckott In the title rendering I don't see the "dot" in ".rprofile..." Can you please check that it will show up?
I'm in airport boarding soon so no time now to do local render, but I plan to do one more check before 5pmish Thurs 12th

Reply:
will check

Reply:
shows up for me